src/services/public_http_server/handlers/relay_webhook/handlers/notify_subscribe.rs

```diff
@@ -177,7 +177,50 @@ pub async fn handle(msg: RelayIncomingMessage, state: &AppState) -> Result<(), R
     };
     info!("Timing: Finished upserting subscriber");
 
-    let notify_topic = subscriber.topic;
+    // TODO do in same txn as upsert_subscriber()
+    if subscriber.inserted {
+        let welcome_notification =
+            get_welcome_notification(project.id, &state.postgres, state.metrics.as_ref())
+                .await
+                .map_err(|e| RelayMessageServerError::NotifyServerError(e.into()))?; // TODO change to client error?
+        if let Some(welcome_notification) = welcome_notification {
+            info!("Welcome notification enabled");
+            if welcome_notification.enabled && scope.contains(&welcome_notification.r#type) {
+                info!("Scope contains welcome notification type, sending welcome notification");
+                let notification = upsert_notification(
+                    Uuid::new_v4().to_string(),
+                    project.id,
+                    Notification {
+                        r#type: welcome_notification.r#type,
```

Comment:
I'm curious about why this field is named like that.

Reply:
`type` is a reserved keyword in Rust so you have to use a [raw identifier](https://doc.rust-lang.org/reference/identifiers.html#raw-identifiers)

Reply:
Ah makes sense. I've normally used kind instead of type when defining attributes

Reply:
This is the serialized name, so I'd have to use serde rename attribute instead